audit-service/src/main/java/uk/gov/gchq/palisade/service/audit/request/common/domain/User.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.palisade.service.audit.request.common.domain;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.util.Objects.requireNonNull;
+
+public class User implements IUser {
+
+    @JsonProperty("user_id")
```

Comment:
Consistency would suggest that this should be "userId"

Reply:
Should probably be kept in the constuctor for consistency

Reply:
Have remove all references to the prototype version of the new User.  Will make note of this suggestion and a few other points made of the design for when its time to re-introduce it back in.